### ISSUE TITLE
Temporarily crashed with simple request mistake.

### ISSUE THREAD

**Issue by UberMiguel** [evepraisal/go-evepraisal#171]
TL;DR: If a user searches for an item based on a copy from their market listing history like the following, the website will stop working for them for many minutes:

Completed	2023.06.17 14:28:00	Intact Shield Emitter	0 / 15	826,900.00 ISK	Stacmon V - Moon 9 - Federation Navy Assembly Plant	Placid

I was just being lazy but something like this should be prevented from disrupting service. I figure enter this on a test server to see what happens and account for such an error to be ignored with a formatting error msg instead of what it currently does. I would think this applies to any other such unrecognizable strings?
